common/src/main/java/whocraft/tardis_refined/common/blockentity/door/RootShellDoorBlockEntity.java

```diff
@@ -12,8 +16,14 @@ public RootShellDoorBlockEntity(BlockPos blockPos, BlockState blockState) {
         super(BlockEntityRegistry.ROOT_SHELL_DOOR.get(), blockPos, blockState);
     }
 
+    @Override
```

Comment:
If it does nothing, do not have it here
If you are removing vanilla functionality, leave a comment

Reply:
Comment left.